### PR TITLE
Add more functions or contribute ideas 

### PR DESCRIPTION
**Pull request by suparna13**
Create function to return third element
Applicable to iterable data types only viz string, list, tuple, etc
Handled exception for non iterable data types.

### CODE REVIEW

**Comment by suparna13** [comment]
@JasonTurley  Can you please take a look at PR. Thank you.

File: three/three.py [hunk 1]
```diff
@@ -72,6 +72,12 @@ def decimal_places(i):
 def days_ago(timestamp):
     return timestamp - (86400 * 3)
 
+def third_element(items):
+	try:
+		return items[2]
```
Comment (buckley-w-david) [inline three/three.py]:
```suggestion
		return items[three()-1]
```

Comment (JasonTurley) [reply three/three.py]: I like the suggestion from @buckley-w-david. 
@suparna13 please add and verify a test case and I will merge.

Comment (JasonTurley) [reply three/three.py]: See the new [contributing guide](https://github.com/JasonTurley/three/blob/master/CONTRIBUTING.md) if unsure on any steps.

Comment (suparna13) [reply three/three.py]: @JasonTurley Added requested changes and test case as well.

**Comment by suparna13** [comment]
@JasonTurley Please take a look at PR. Updated suggested changes and added test case.

**Review by JasonTurley — APPROVED** [review]
Looks good.

**Pull request merged**